roles/hosted_engine_setup/tasks/filter_unsupported_devices.yml

```diff
@@ -22,14 +22,17 @@
   set_fact:
     team_if: "{{  team_list.results | reject('skipped') | map(attribute='nic_if.nic') | list }}"
 - debug: var=team_if
+- import_tasks: filter_unsupported_vlan_devices.yml
 - name: Filter unsupported interface types
   set_fact:
-    otopi_host_net: "{{ host_net | difference(team_if) }}"
+    otopi_host_net: "{{ host_net | difference(invalid_net_if) }}"
   register: otopi_host_net
 - debug: var=otopi_host_net
-- name: Failed if only teaming devices are available
+- name: Failed if only unsupported devices are available
   fail:
     msg: >-
-      Only Team devices {{ team_if | join(', ') }} are present.
+      Only unsupported devices {{ invalid_net_if | join(', ') }} are present.
       Teaming is not supported.
+      VLAN devices with bond as base interface configured with unsupported bond mode is not supported.
```

Comment:
The message is not clear, maybe change it to something like:
Only unsupported devices {{ invalid_net_if | join(', ') }} are present.
Teaming and bond modes: Round Robin, TLB, ALB  are unsupported.
The following bond modes are supported: {{ acceptable_bond_modes }}

Please create a variable for the supported bond list and use it also in `filter_unsupported_vlan_devices.yml`